how-to/setup_new_users.rst

```diff
@@ -0,0 +1,69 @@
+JAAS: Setup New Users
+=====================
+
+This how-to describes how a new user can begin to use JAAS to deploy applications.
+
+Prerequisites
+-------------
+
+For this how-to you will need the following:
+
+- A running JAAS environment, see :doc:`our tutorial <../tutorial/deploy_jaas_microk8s>`.
+- A controller connected to JAAS, see :doc:`our how-to <./add_controller>`. This how-to assumes you have added an LXD controller.
+
+Login to JAAS
+-------------
+
+Login to JAAS for the first time:
+
+.. code:: bash
+
+    juju login test-jimm.localhost:443 -c jaas
```

Comment:
Perhaps we should use more realistic DNS names and not "test-jimm" and "localhost"?